ui/raidboss/data/triggers/o5n.js

```diff
@@ -144,9 +159,9 @@
       replaceSync: {
         'Agony': 'Fantôme Souffrant',
         'Doom Chimney': 'Cheminée Maléfique',
-        'Phantom Train': 'Train Fantôme',
+        'Phantom Train': 'Train fantôme',
```

Comment:
Oh no, is it lowercase f in game? Just curious for future translations so that I can attempt to do this right.  This is what fflogs gave me as the mob name.

Reply:
Well in the log file, it is lower case :
03|2018-06-05T22:11:17.1780000+02:00|4000440e|Train fantôme|0|46|dfa2|2ee0|0|a3c54fd19768a98f2fcc24275375ef64

Reply:
Yeah, I figured that might be it.  Thanks!  I don't know how to figure out which mobs should be upper or lowercase, so I guess you'll have to just correct it to lowercase when it ends up being wrong.

Reply:
actually sometime it can be both, that's why I put 'Wroth Ghost': 'Fantôme (F|f)urieux' in replaceSync 

in the timeline :
8 "--sync--" sync /:28AE:Wroth Ghost/ window 8,3
13 "Encumber" sync /:Wroth Ghost:28AE:/ 

directly in ACT (right-click->view logs):
[23:54:34.149] 14:28AE:Fantôme Furieux starts using Encombrement on Fantôme Furieux.
[23:54:38.123] 16:400156BF:Fantôme furieux:28AE:Encombrement:E0000000::0:0:0:0:0:0:0:0:0:0:0:0:0:0:0:0::::::::::57250:57250:12000:12000:1000:1000:0:17.5:0.02499723:
 
in ACT network log:
20|2018-06-07T23:54:34.1490000+02:00|400156BF|Fantôme furieux|28AE|Encombrement|400156BF|Fantôme furieux||83a71dbd36c81d699a85bfce066129f8
22|2018-06-07T23:54:38.1230000+02:00|400156BF|Fantôme furieux|28AE|Encombrement|E0000000||0|0|0|0|0|0|0|0|0|0|0|0|0|0|0|0||||||||||57250|57250|12000|12000|1000|1000|0|17.5|0.02499723||b44129ea2469ff5dbf96a1ab9a7b59dc

I don't know why it's this way, and I don't know which log Cacbot use

Reply:
Oh, that's quite weird! I'm pretty sure cactbot uses the former (the logs that go to ACT), but I don't have any easy examples other than this that test one way or the other.